src/t_zset.c

```diff
@@ -2275,6 +2275,16 @@ int zuiFind(zsetopsrc *op, zsetopval *val, double *score) {
             } else {
                 return 0;
             }
+        } else if (op->encoding == OBJ_ENCODING_LISTPACK) {
+            zuiSdsFromValue(val);
+            unsigned char *lp = op->subject->ptr;
+            unsigned char *p = lpFirst(lp);
+            if (p && lpFind(lp, p, (unsigned char*)val->ele, sdslen(val->ele), 0)) {
```

Comment:
It's embarrassing that I missed this else if on the encoding of a set.

This looks good, but I think there is no reason for zuiFind to go into the internals of the set. It can simply use `setTypeIsMember` and don't care about encoding.

Reply:
good idea, the code is much simpler!